web-admin/src/components/home/ProjectList.svelte

```diff
@@ -11,10 +14,24 @@
   <ol>
     {#each $projs.data.projects as proj}
       <li class="ml-2">
-        <h3 class="text-gray-500 font-semibold mb-1" style="font-size: 10px;">
-          {proj.name.toUpperCase()}
-        </h3>
-        <DashboardList {organization} project={proj.name} />
+        <Tooltip location="left" distance={4}>
+          <a
+            class="flex max-w-fit items-center gap-x-1 mb-1 hover:underline hover:text-gray-700"
+            href="{organization}/{proj.name}/-/deployment"
+          >
+            <DeploymentStatusChip {organization} project={proj.name} />
+            <h3 class="text-gray-500 font-semibold" style="font-size: 10px;">
```

Comment:
Should it use Tailwind for font sizes?

Reply:
Default Tailwind theme doesn't go below 12px. Now that Janet has started we should [customize the theme](https://tailwindcss.com/docs/font-size#customizing-your-theme), but need to be considerate of Rill Developer when doing so.

I suppose I should have used Tailwind [arbitrary values](https://tailwindcss.com/docs/font-size#arbitrary-values).